src/Sulu/Bundle/AdminBundle/Resources/js/components/Snackbar/snackbar.scss

```diff
@@ -8,15 +8,20 @@ $snackbarWarningColor: $blueZodiac;
 $snackbarWarningBackgroundColor: $gold;
 
 .snackbar {
+    position: relative;
+    z-index: 2;
     height: $snackbarHeight;
     padding: 0 20px;
     display: flex;
     align-items: center;
     will-change: margin-top;
-    transition: margin-top .15s;
+    transition: margin-top .15s, z-index 0s .15s;
```

Comment:
why do we need a z-index now? and why do we animate it? 🤔

Reply:
same for visibility

Reply:
It's not really animated, the `z-index` and `visibility` are only delayed. The thing is that we have two snackbars (one for warnings, one for errors), and they are interferring with each other.

The `z-index` is to make sure that the currently visible one lies over the one not being visible (otherwise the warning snackbar will slide over the error snackbar if both were visible, and appear after it has blocked the sight to the error snackbar).

The visibility also makes sure that the snackbar is not visible if it was e.g. moved down by a snackbar before itself (since we are animating using `margin-top` that can happen if the error toolbar is visible, but the warning is not).

And these two values are not really animated, they are "only" delayed (that's the second time value in the property, not the first one, which is the animation duration). And that's because otherwise the snackbar will be made invisible before it started moving up again when being closed.